api/src/constants/form.js

```diff
@@ -60,6 +80,10 @@ export const REQUEST_TO_FORM_CONTENT = {
   forecastAmount: 'bceid.forecastAmount',
   prodDate: 'bceid.prodDate',
   useDate: 'bceid.useDate',
+  // v2:
+  appName: 'bceid.appName',
+  contactInfo: BCEID_CONTACT_INFO_SCHEMA(['es', 'pm', 'tl', 'pl', 'sl', 'cl']),
```

Comment:
are those values constant? (es, pm, tl, pl, sl, cl) what do they mean? where else can they be found?